data/na/swagger/1-0-5.yaml

```diff
@@ -35,6 +35,8 @@ paths:
         payment data for card, cash, cheque, profile, and token. You must change the payment&#95;method for each payment type. Credit Card - "card",
         Payment Profile - "payment&#95;profile", Single Use Token - "token", Cash - "cash", Cheque - "cheque", Apple Pay - "apple&#95;pay",
         Google Pay - "google&#95;pay".
+
+        Please note Cardholder name and either Billing Email or Billing Phone Number are **mandatory** for Visa. 
```

Comment:
If "Billing Email" and "Billing Phone Number" use capital for first letter, shall we make the "Cardholder name" as "Cardholder Name" ?

Reply:
Good idea, I changed it